_includes/share-meta.html

```diff
@@ -1,17 +1,19 @@
 {% capture title %}{% if page.title %}{{ page.title }}{% else %}{{ site.name }}{% endif %}{% endcapture %}
 {% capture description %}{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines }}{% else %}{{ site.description }}{% endif %}{% endcapture %}
+{% capture img %}{% if page.image %}{{ page.image }}{% else %}{{ site.thumbnail_path }}{% endif %}{% endcapture %}
```

Comment:
Do we set `page.image` on the deployment guides?

Reply:
Yes each deployment guide has an image that can be accessed with `page.image`